FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MultiComputable.java

```diff
@@ -3,4 +3,8 @@
 public interface MultiComputable extends StatisticsReportable {
     float[] multiCompute(float[] values);
     String getOutputUnits();
+
+    default int getStatCount() {
```

Comment:
Do the classes that implement this interface 'know' the statCount when they are created?

Reply:
When the class that implements the MutiComputable  interface is instantiated (such as Percentiles), the number of statistics is passed to the constructor and stored in an array (for percentiles, the number of percentiles being computed is stored as a float array).  The getStatCount will iterate through the array to see how many statistics are stored.